shared/src/packages/users/libs/types/business-sign-up-response-dto.type.ts

```diff
@@ -1,5 +1,15 @@
+import { type BusinessEntityT } from '~/packages/business/business.js';
+
 import { type UserEntityT } from './user-entity.type.js';
 
-type BusinessSignUpResponseDto = Pick<UserEntityT, 'id' | 'phone'>;
+// import { type UserGroupEntityT } from './user-group-entity.type.js';
+
+//temporary type!!!
```

Comment:
in Shared scope we have type UserGroupEntityT = { id: number; name: UserGroupNameT; key: UserGroupKeyT; }
but it is not suitable, because groupService returns in type GroupEntityObjectT =  { id: number; name: string; key: string; }. But I can't use it there because it exist in backend only and it is impossible to import it in shared.
I thought that use UserGroupEntityT is better to use and even started to correct, but there are a lot of changes to do in someone's code.
So I desided to pull this for now, and then deal with this type.

Reply:
upd
I've create type UserGroupEntityObject = { id: number; name: string; key: string; }; in Shared and used it.
But I still think using type  { id: number; name: UserGroupNameT; key: UserGroupKeyT; } would be better. But it needs to make a lot of changes in groupService and businessService.